lib-ce/emqx_modules/test/emqx_mod_delayed_SUITE.erl

```diff
@@ -76,3 +76,42 @@ t_delayed_message(_) ->
     EmptyKey = mnesia:dirty_all_keys(emqx_mod_delayed),
     ?assertEqual([], EmptyKey),
     ok = emqx_mod_delayed:unload([]).
+
+t_delayed_with_acl(_) ->
+    ok = emqx_mod_delayed:load([]),
+    Case = fun() ->
+        {ok, C1} = emqtt:start_link([{clean_start, true}, {proto_ver, v5}]),
+        {ok, _} = emqtt:connect(C1),
+        emqtt:publish(C1, <<"$delayed/2/allow/1">>, <<"allow">>, []),
+        emqtt:publish(C1, <<"$delayed/2/deny/1">>, <<"deny">>, []),
```

Comment:
publish the `"deny"` message first?

Reply:
NO. `deny_action` is by default `ignore`. and I don't see any specific setting in this test case.